app/components/preprint-form-authors.js

```diff
@@ -10,26 +10,158 @@ export default CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
     permissionChanges: {},
     bibliographicChanges: {},
     valid: Ember.computed.alias('newContributorId'),
+    permissionToggle: false,
+    bibliographicToggle: false,
+    removalToggle: false,
+    stillAdmin: Ember.computed('isAdmin', function() {
+        return this.get('isAdmin');
+    }),
+    addState: 'emptyView', // There are 3 view states on left side of Authors panel. Default state just shows search bar.
+    query: null,
+    // Total contributor search results
+    totalSearchResults: Ember.computed('searchResults.[]', function() {
+        let searchResults = this.get('searchResults');
+        if (searchResults && searchResults.links) {
+            return Math.ceil(searchResults.links.meta.total);
+        } else {
+            return;
+        }
+    }),
+    // Total pages of contributor search results
+    pages: Ember.computed('totalSearchResults', function() {
+        let totalSearchResults = this.get('totalSearchResults');
+        return Math.ceil(totalSearchResults / 10);
```

Comment:
Avoid hardcoding an assumption that pages always have 10 results.

Actually, we needed a field for "number of pages" elsewhere, so it should already be calculated and present under meta.total.

(The names are confusing. Sorry.)

https://github.com/abought/ember-osf/blob/7595c72f52b193d66fc2c00ffe5102a1c44ded22/addon/serializers/osf-serializer.js#L120-L120